### ISSUE TITLE
Rules MD041 and MD033 aren't compatible with default configuration

### ISSUE THREAD

**Issue by lorenzogrv** [DavidAnson/markdownlint#426]
As per example at your documentation of rule [MD041](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md041), the following should work:

```markdown
<h1 align="center"><img src="https://placekitten.com/300/150"/></h1>

This is a file with a top-level HTML heading
```

But that example doesn't work as the rule [MD033](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md041) does not allow inline html.

Shouldn't it work with the default configuration values?

**Comment by nschonni** [comment]
The default would be to use a `#` for an H1 or a frontmatter title, so there is not conflict with the default configuration.

**Comment by lorenzogrv** [comment]
@nschonni , @DavidAnson Thanks for clarification. Will close as I consider it is resolved.
